pyproject.toml

```diff
@@ -151,9 +151,9 @@ docs = [
     # for notebooks in the gallery
     "MEArec",   # Use as an example
     "datalad==0.16.2",  # Download mearec data, not sure if needed as is installed with conda as well because of git-annex
-    "pandas", # Don't know where this is needed
-    "hdbscan>=0.8.33",   # For sorters, probably spikingcircus
-    "numba", # For sorters, probably spikingcircus
+    "pandas", # in the modules gallery comparison tutorial
+    "hdbscan>=0.8.33",   # For sorters spykingcircus2 + tridesclous
+    "numba", # For many postprocessing functions
```

Comment:
Thanks a bunch for adding those comments. I added the others. 

Concerning numba, are they used to generate the examples on quality metrics?

Reply:
https://spikeinterface.readthedocs.io/en/latest/how_to/get_started.html
uses the post processing functions. This is just a written out tutorial, but sphinx can be weird at import, so I'm also still not sure where exactly it is required, but I know the post processing module requires it. Quality metrics are a possibility, since numba is used for the sliding rp violation. I didn't see that as a specific example used, but (there are a lot of docs :) ) . Tridesclous2 also uses numba, but again the examples use Tridesclous ( and I didn't check Tridesclous source for this). 

So I think between the two of us we will eventually get the comments right!

Reply:
Great. Thanks for the improvements : )